modules/virt-installing-watchdog-agent.adoc

```diff
@@ -12,6 +12,15 @@ You install the watchdog agent on the guest and start the `watchdog` service.
 
 . Log in to the virtual machine as root user.
 
+. This step is only required when installing on {ibm-z-name} (´s390x`). Enable `watchdog` by running the following command:
++
+[source,terminal]
+----
+# modprobe diag288_wdt
+----
++
+A message is displayed, showing that the `/dev/watchdog` file path is now present in the VM.
```

Comment:
this wont be displayed when we run modprobe diag288_wdt we have to explicitly check so may be we can have one more step 
1. Verify /dev/watchdog is present in the vm
      ls /dev/watchdog